scripts/generateUsersActivityReportCsv.js

```diff
@@ -0,0 +1,287 @@
+/**
+ * NODE_ENV=beta node ./scripts/generateUsersActivityReportCsv.js
+ */
+
+const { writeFileSync } = require('fs');
+const config = require('config');
+const { Parser } = require('json2csv');
+const mongoose = require('mongoose');
+
+const appFactory = () => {
+  const data = {};
+  return {
+    get(key) {
+      return data[key];
+    },
+    set(key, val) {
+      data[key] = val;
+    },
+  };
+};
+
+const app = appFactory();
+app.set('mongooseClient', mongoose);
+const aggregteQuery = [
+  {
+    $match: {
+      giverId: {
+        $exists: true,
```

Comment:
This condition will show users with giverId of zero. Like this one
https://feathers.beta.giveth.io/users/0x97B30e465eB68a4a3333D2142a386D7e20DbE195

This kind of users should be ignored.

Reply:
yeah but some users with `giverId:0` have activity indeed, like this one
https://feathers.beta.giveth.io/users/0x51e37e3716c0ED418a90e519CeEE395d8f6deCB9

Reply:
You are right

Reply:
@mohammadranjbarz so this one is the latest output?
https://docs.google.com/spreadsheets/d/1tIDLibzAF3g7dsilEH_yakuExbXHoWUNZGdcG0Lez4k/edit?usp=sharing

Reply:
@aminlatifi 
Yes it's the last one, 
but if you want some changes I think it's faster to apply the changes in a 1-1 meeting

Reply:
> @mohammadranjbarz so this one is the latest output?
> https://docs.google.com/spreadsheets/d/1tIDLibzAF3g7dsilEH_yakuExbXHoWUNZGdcG0Lez4k/edit?usp=sharing

@willyogo @bowensanders You can see the latest version in this link